addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1680,16 +1681,19 @@ const Wysiwyg = Widget.extend({
         if (isInMedia || (link && link.isContentEditable)) {
             // Handle the media/link's tooltip.
             this.showTooltip = true;
-            setTimeout(() => {
+            this.tooltipTimeouts.push(setTimeout(() => {
                 // Do not show tooltip on double-click and if there is already one
                 if (!this.showTooltip || $target.attr('title') !== undefined) {
                     return;
                 }
                 this.odooEditor.observerUnactive();
-                $target.tooltip({title: _t('Double-click to edit'), trigger: 'manual', container: 'body'}).tooltip('show');
+                // Tooltips need to be cleared before leaving the editor.
+                this.saving_mutex.exec(() => {
+                    $target.tooltip({title: _t('Double-click to edit'), trigger: 'manual', container: 'body'}).tooltip('show');
```

Comment:
The problem didn't show up before because if the target doesn't exist, it probably means that the page was reloaded. But here we were in a situation where in the middle of showing a tooltip, the page would reload, and produce a traceback. (The test clicks on the image, changes it, clicked on save and the traceback related to tooltips would appear.)

It used to be consistent on this branch and would prevent us from passing the test `:TestUiPublisher.test_course_publisher_elearning_manager`

But I just tried commenting out the code and my branch was green 🤷🏻 so I'm not sure if we should keep it.

Should I remove it?